android/cli/commands/_build.js

```diff
@@ -879,7 +879,11 @@ AndroidBuilder.prototype.validate = function validate(logger, config, cli) {
 	cli.tiapp.properties['ti.deploytype'] = { type: 'string', value: this.deployType };
 
 	// Fetch Java max heap size setting.
-	this.javacMaxMemory = cli.tiapp.properties['android.javac.maxmemory'] && cli.tiapp.properties['android.javac.maxmemory'].value || config.get('android.javac.maxMemory', '3072M');
+	if (cli.tiapp.properties['android.javac.maxmemory'] && cli.tiapp.properties['android.javac.maxmemory'].value) {
+		logger.error(__('android.javac.maxmemory is deprecated. Please use android.javac.maxMemory') + '\n');
+		process.exit(1);
```

Comment:
I'm not sure how much work extra it might be, but given that this is deprecation rather than straight removal initially, maybe warning and continuing is better here?

Reply:
Then the user won't see it as it is right in the beginning of the output. Had that first and it passed by so quickly :smile: 
But I'm happy to change it to a warning, accept both values and we remove it in a later SDK :+1:

Reply:
I've kept it as an error so it will better to see. But I accept both values now and print the error (without exit) when you use `maxmemory`.